src/components/Calendar/ExportCalendar.js

```diff
@@ -250,6 +251,7 @@ class ExportCalendarButton extends PureComponent {
                 // Download the .ics file
                 var blob = new Blob([val], { type: 'text/plain;charset=utf-8' });
                 saveAs(blob, 'schedule.ics');
+                openSnackbar('success', 'Schedule downloaded! Make sure your calendar is in PST.', 5);
             } else {
                 console.log(err);
```

Comment:
Since you're working here, let's also add a snackbar for when it errors. Right now it just silently fails.
```suggestion
                openSnackbar('error', 'Something went wrong! Unable to download schedule.', 5);
                console.log(err);
```

Reply:
(This can be tested by trying to download a summer schedule)

Reply:
Also feel free to put whatever you want for the error message, that's just a suggestion.